device-registry/src/main/scala/org/genivi/sota/device_registry/DevicesResource.scala

```diff
@@ -197,6 +207,11 @@ class DevicesResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
     } ~
     (path("packages" / "affected") & scope.post) {
       findAffected(authedNs)
+    } ~
+    pathPrefix("package_stats") {
```

Comment:
It is a bit sad that everything in the `devicePackagesRoutes` have different prefixes. Probably something we want to clean up later.